openff/toolkit/tests/test_utils_serialization.py

```diff
@@ -118,6 +120,40 @@ def test_pickle(self):
         assert self.thing == thing_from_pkl
 
 
+class TestNumPySerialization:
+    @pytest.mark.parametrize(
+        "dtype",
+        [
+            (np.dtype(float).newbyteorder(">")),
+            (np.dtype(float).newbyteorder("<")),
+        ],
+    )
+    def test_serialize_endianness(self, dtype):
+        """Test that arrays are serialized as big-endian"""
+        # Force big- (">") or little-endianness ("<") , overriding architecture default
+        arr = np.arange(3).astype(float).astype(dtype)
+
+        # This bytestring should be a big-endian representation, generated by
+        # np.arange(3).astype(np.dtype(float).newbyteorder(">")).tobytes()
+        assert serialize_numpy(arr)[0] == (
+            b"\x00\x00\x00\x00\x00\x00\x00\x00?\xf0\x00\x00\x00\x00\x00\x00@\x00\x00\x00\x00\x00\x00\x00"
+        )
```

Comment:
```suggestion
    @pytest.mark.parametrize("endian", [">", "<"])
    def test_serialize_endianness(self, endian):
        """Test that arrays are serialized as big-endian"""
        # Force big- (">") or little-endianness ("<") , overriding architecture default
        dtype = np.dtype(float).newbyteorder(endian)
        arr = np.arange(3).astype(dtype)

        # This bytestring should be a big-endian representation, generated by
        # np.arange(3).astype(np.dtype(float).newbyteorder(">")).tobytes()
        assert serialize_numpy(arr)[0] == (
            b"\x00\x00\x00\x00\x00\x00\x00\x00?\xf0\x00\x00\x00\x00\x00\x00@\x00\x00\x00\x00\x00\x00\x00"
        )
```

Maybe just passing the endian direction would save some vertical space?